cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/type/impl/CassandraYesNoType.java

```diff
@@ -21,7 +20,7 @@
 	public static final CassandraYesNoType INSTANCE = new CassandraYesNoType();
 
 	public CassandraYesNoType() {
-		super( StringMappedGridTypeDescriptor.INSTANCE, org.hibernate.type.YesNoType.INSTANCE.getJavaTypeDescriptor() );
+		super( CharacterStringGridTypeDescriptor.INSTANCE, org.hibernate.type.YesNoType.INSTANCE.getJavaTypeDescriptor() );
```

Comment:
Could you check whether this one works alternatively:

    super( new TranslatingGridTypeDescriptor( Character.class ), org.hibernate.type.YesNoType.INSTANCE.getJavaTypeDescriptor() );

Would save that new grid descriptor type. The whole type business is far from perfect btw., it's very high on our agenda to rework that. So many thanks for fighting through it :)